### PR TITLE
Migrating to redux(Auth page)

### PR DESCRIPTION
**Pull request by spumony**
- added gitignore file
- edited Adeonmaster's issues

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (AdeonMaster) [inline .gitignore:1]: Add package lock file here too

File: src/components/Logout/Logout.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    this.props.logout()
+  }
+
+  render() {
+    return <Redirect to={'/'} />
+  }
+}
+
+
+function mapDispatchToProps(dispatch) {
```
Comment (AdeonMaster) [inline src/components/Logout/Logout.js]: const mapDispatchToProps = { logout}

File: src/containers/Auth/Auth.js [hunk 3]
```diff
@@ -146,19 +136,27 @@ export default class Auth extends Component {
...
           </form>
         </div>
       </div>
     )
   }
-}
\ No newline at end of file
+}
+
+function mapDispatchToProps(dispatch) {
```
Comment (AdeonMaster) [inline src/containers/Auth/Auth.js]: Simplify this

File: src/containers/QuizCreator/QuizCreator.js [hunk 4]
```diff
@@ -171,15 +172,15 @@ class QuizCreator extends Component {
   }
 }
 
-function mapStateToProps(state) {
+const mapStateToProps = state => {
```
Comment (AdeonMaster) [inline src/containers/QuizCreator/QuizCreator.js]: Simplify this

File: src/containers/QuizCreator/QuizCreator.js [hunk 5]
```diff
@@ -171,15 +172,15 @@ class QuizCreator extends Component {
...
   return {
     quiz: state.create.quiz
   }
 }
 
-function mapDispatchToProps(dispatch) {
+const mapDispatchToProps = dispatch => {
```
Comment (AdeonMaster) [inline src/containers/QuizCreator/QuizCreator.js]: Simplify

**Review by AdeonMaster — CHANGES_REQUESTED** [review]

File: src/containers/QuizCreator/QuizCreator.js (hunk 4 above)
Comment (spumony) [reply src/containers/QuizCreator/QuizCreator.js]: how?

Comment (AdeonMaster) [reply src/containers/QuizCreator/QuizCreator.js]:
```
const mapDispatchToProps = dispatch => {
  return {
    createQuizQestion: item => dispatch(createQuizQuestion(item)),
    finishCreateQuiz: () => dispatch(finishCreateQuiz())
  }
}
```
=>
```
const mapDispatchToProps = {
createQuizQestion,
finishCreateQuiz,
};
```

File: firebase.json [hunk 6]
```diff
@@ -0,0 +1,16 @@
+{
+  "hosting": {
```
Comment (AdeonMaster) [inline firebase.json]: You should combine this file with .firebaserc so .firebaserc will look like:

```
{
  "projects": {
    "default": "react-quiz-468e0"
  },
  "hosting": {
    "public": "build",
    "ignore": [
      "firebase.json",
      "**/.*",
      "**/node_modules/**"
    ],
    "rewrites": [
      {
        "source": "**",
        "destination": "/index.html"
      }
    ]
  }
}
```

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1,4 @@
...
+build
+.firebase
+package-lock.json
```
Comment (AdeonMaster) [inline .gitignore:4]: Empty line at the end please

**Pull request merged**